third_party/move/move-vm/runtime/src/loader/mod.rs

```diff
@@ -291,7 +291,7 @@ impl Loader {
     pub(crate) fn load_script(
         &self,
         script_blob: &[u8],
-        ty_args: &[TypeTag],
+        ty_arg_tags: &[TypeTag],
```

Comment:
Probably not worth to emphasize the fact that these args are represented as tags here.

Reply:
Changed back to types.